source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,142 @@
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.42.1
+
+You can create Atlas Vector Search indexes using |compass|. These indexes 
+enable you to index vector data *and* other data types, facilitating 
+semantic searches on the indexed fields. Atlas Vector Search indexes support 
+indexing vector data by identifying the most similar vectors. This is achieved 
+by calculating the distance between the query vector and the vectors stored 
+in the index.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be either:
+
+- Hosted on `MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and
+  have an Atlas cluster tier of M10 or higher.
+
+- A local deployment that is set up using the `Atlas CLI
+  <https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/>`__.
+
+Additionally, your deployment must run MongoDB version 7.0 or later.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <indexes-tab>` tab, click the
+      :guilabel:`Create` button, then click :guilabel:`Search Index`.
+
+   .. step:: Specify a name for the index
+
+   .. step:: Select the Atlas Search Index type
+
+      Select :guilabel:`Vector Search`.
+
+   .. step:: Provide the Atlas Vector Search Index configurations
+
+      You must initially provide the following default vector search index 
+      configurations. You can modify this later.
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 20 10 70
+
+         * - Field
+           - Type
+           - Description
+
+         * - ``path``
+           - string
+           - The field name to index.
+        
+         * - ``numDimensions``
+           - int 
+           - The number of vector dimensions, which Atlas Search enforces at index- and 
+             query-time. This value can't be greater than 2048.
+        
+         * - ``similarity``
+           - string
+           - The vector similarity function used to search for top K-nearest neighbors.
+             Select from the following:
+
+             .. list-table::
+                :header-rows: 1
+                :widths: 50 50
+
+                * - Function
+                  - Description
+
+                * - ``euclidean``
+                  - A function that measures the distance between ends of vectors. This allows 
+                    you to measure similarity based on varying dimensions. 
+
+                * - ``cosine``
+                  - A function that measures similarity based on the angle between vectors. This 
+                    allows you to measure similarity that isn't scaled by magnitude. You can't use 
+                    zero magnitude vectors with cosine. To measure cosine similarity, we recommend 
+                    that you normalize your vectors and use dotProduct instead.
```

Comment:
[suggestion/question] Do you think it would be useful to put the zero magnitude vector limitation in a note as opposed to in the main body of this list item? I'm wondering how often this use case might come up and whether users will notice the limitation if It is not highlighted.

Reply:
updated.